config/default/kustomization.yaml

```diff
@@ -12,27 +12,8 @@ namePrefix: serverless-manager-
 commonLabels:
   app.kubernetes.io/component: serverless-manager.kyma-project.io
 
-# Labels to add to all resources and selectors.
-#labels:
-#- includeSelectors: true
-#  pairs:
-#    someName: someValue
 
 resources:
 - ../crd
 - ../rbac
 - ../manager
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
-# crd/kustomization.yaml
-#- ../webhook
-# [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
-#- ../certmanager
-# [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
-#- ../prometheus
```

Comment:
can we remove [the following](https://github.com/kyma-project/serverless-manager/tree/main/config/prometheus) folder as well, please?

Reply:
done